### ISSUE TITLE
Python errors dont seem to show

### ISSUE THREAD

**Issue by sudokit** [helix-editor/helix#7217]
### Summary

When coding in python, no errors are shown. Code completion works but not errors. Other languages work well

### Reproduction Steps

https://asciinema.org/a/h5xRiUB9YeaPjjtq9RPIwleOz

(archived in 7 days)

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
2023-06-03T19:26:11.197 helix_lsp::transport [ERROR] Tried sending response into a closed channel (id=Num(1)), original request likely timed out
2023-06-03T19:26:11.204 helix_lsp::transport [ERROR] Tried sending response into a closed channel (id=Num(2)), original request likely timed out
```

</details>

### Platform

Linux

### Terminal Emulator

kitty 0.28.1 created by Kovid Goyal

### Helix Version

helix 23.05

**Comment by gabydd** [comment]
Pylsp doesn't have diagnostics out of the box it relies on third party providers for diagnostics see: https://github.com/python-lsp/python-lsp-server#installation if you want to use one that is disabled by default (flake8 instead of pyflakes) you have to manually enable it in your config

**Comment by sudokit** [comment]
Right, okay thanks! And wouldn't it be better to use something like pylance in that case?

**Comment by gabydd** [comment]
pylance is a proprietary language server and we would be violation of the license if we used it with helix: https://github.com/microsoft/pylance-release/blob/main/FAQ.md#can-i-load-pylance-in-code--oss
